src/components/Share.tsx

```diff
@@ -37,6 +43,7 @@ const Share = ({ url, title, twitterHandle }) => {
       <span>Share article</span>
       <TwitterShareButton
         url={url}
+        // @ts-expect-error quote doesn't exist in TwitterShareButton
         quote={title}
```

Comment:
Let me know if I can remove it (as `TwitterShareButton` doesn't use it)

Reply:
Sure thing!